engine/src/main/java/org/pentaho/di/trans/steps/dimensionlookup/DimensionLookupMeta.java

```diff
@@ -1360,7 +1359,7 @@ private void checkLookup( List<CheckResultInterface> remarks, StepMeta stepinfo,
 
         if ( !Utils.isEmpty( tableName ) ) {
           String schemaTable = databaseMeta.getQuotedSchemaTableCombination( schemaName, tableName );
```

Comment:
Don't think we need this line anymore except for error condition. Maybe only run that code in case of error.